### ISSUE TITLE
lodash/collection-return false positive

### ISSUE THREAD

**Issue by lsentkiewicz** [wix/eslint-plugin-lodash#131]
```js

const promises = _.map(items, async (items) => {
  await foo();
  await bar();
  // don't return anything
});

Promise.all(promises);
```

`promises` is an array of promises because  an `async` lambda returns always a promise

**Comment by ganimomer** [comment]
Fixed in `v2.3.2`. Thanks for reporting!
